### ISSUE TITLE
[enhancement] Add option to specify targets from file

### ISSUE THREAD

**Issue by jiayuqi7813** [p0dalirius/FindUncommonShares#27]
The DC is close ldap，cant get computers，but I have computers ip list in IP.txt，How do I operate

**Comment by CaledoniaProject** [comment]
The current scan target is populated from the ldap query. Need an cli argument to load the IP or FQDN from a file, one per line.

The scenario is, in a large corporation there could be tons of unwanted targets, it would be almost impossible to create a suitable LDAP query to filter out the targets. So it's best to let the users to make the choice.

**Comment by p0dalirius** [comment]
Hi,

New features are now implemented in release 3.2 (753e277)

If you want to scan only one IP without using LDAP:
```
./FindUncommonShares.py -tt 192.168.1.12 -au "$USER" -ap "$PASSWORD" -ai "$DC_IP" -ns $DC_IP --no-ldap
```

If you want to scan only multiple IPs without using LDAP:
```
./FindUncommonShares.py -tt 192.168.1.12 -tt 192.168.1.13 -au "$USER" -ap "$PASSWORD" -ai "$DC_IP" -ns $DC_IP --no-ldap
```


If you want to scan only multiple IPs ir CIDR ranges from a file without using LDAP:
```
./FindUncommonShares.py -tf targets.txt -au "$USER" -ap "$PASSWORD" -ai "$DC_IP" -ns $DC_IP --no-ldap
```

Let me know if you encounter any problem,

Best regards,
